### DISCUSSION TITLE
Welcome to Reqnroll Discussions!

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Technology19** [comment]


**Reply by gasparnagy** [reply]
@Technology19 your question has been answered, so I remove this comment from here, because it is at the wrong topic anyway. For further migration questions, please create a new topic at https://github.com/orgs/reqnroll/discussions/categories/specflow-migration-q-a
